scripts/restore-sanitised-database.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Function to list and select an AWS Vault profile
+select_aws_vault_profile() {
+    local prompt_message=$1
+    local profile_variable=$2
+
+    echo "$prompt_message"
+    PS3='Please select a profile (or Ctrl-C to quit): '
+    IFS=$'\n' read -r -d '' -a profiles <<< "$(aws-vault list --profiles)"
+    select profile in "${profiles[@]}"; do
+        if [[ -n "$profile" ]]; then
+            echo "Selected profile: $profile"
+            export "$profile_variable"="$profile"
+            break
+        else
+            echo "Invalid selection, please try again."
+        fi
+    done
+}
+
+# Prompt user for AWS profiles based on environment
+select_aws_vault_profile "Select the AWS Vault profile for the ${ENV} environment:" AWS_VAULT_PROFILE_TO_SANITISE
```

Comment:
How is `AWS_VAULT_PROFILE_TO_SANITISE` getting defined?

Reply:
select_aws_vault_profile funtion is taking two args prompt_message and the profile_variable where we would store our profile, In our case it's AWS_VAULT_PROFILE_TO_SANITISE. This line 
'export "$profile_variable"="$profile" is setting profile to our profile_variable (AWS_VAULT_PROFILE_TO_SANITISE). I hope it make sense.